modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/actions/types.js

```diff
@@ -15,6 +15,7 @@
 export const CREATE_SEGMENTS_EXPERIENCE = 'CREATE_SEGMENTS_EXPERIENCE';
 export const SELECT_SEGMENTS_EXPERIENCE = 'SELECT_SEGMENTS_EXPERIENCE';
 export const DELETE_SEGMENTS_EXPERIENCE = 'DELETE_SEGMENTS_EXPERIENCE';
+export const DUPLICATE_SEGMENTS_EXPERIENCE = 'DUPLICATE_SEGMENTS_EXPERIENCE';
```

Comment:
👌  thanks for the heads up! I can try to rebase this PR with the one already sent

Reply:
It's a good idea because when this one is ready to forward, we can do it normally and there will be no conflics :) As @cgoncas said, you just have to mention "on top of...". Thanks!

Reply:
can you share the PR to rebase with?

Reply:
https://github.com/liferay-tango/liferay-portal/pull/774